django_project/frontend/src/containers/MainPage/Metrics/ActivityDonutChart.tsx

```diff
@@ -0,0 +1,143 @@
+import React, { useEffect, useState } from "react";
+import { Box, Typography } from "@mui/material";
+import { Doughnut } from "react-chartjs-2";
+import { CategoryScale } from "chart.js";
+import Chart from "chart.js/auto";
+import ChartDataLabels from "chartjs-plugin-datalabels";
+import "./index.scss";
+import axios from "axios";
+import Loading from "../../../components/Loading";
+
+
+Chart.register(CategoryScale);
+Chart.register(ChartDataLabels);
+
+const FETCH_ACTIVITY_PERCENTAGE_URL = '/activity-percentage/'
+
+const ActivityDonutChart = () => {
+    const [loading, setLoading] = useState(false)
+    const [activityData, setActivityData] = useState([])
+    const activityType = {
+        "Unplanned/illegal hunting": "#FF5252",
```

Comment:
I understand that they wanted this to be static, but let's implement that in the backend instead. This way, we won't need to write the activity type twice.

Reply:
@dimasciput Should I send all the activity_type with activity_type's colour from the back-end?

Reply:
@dimasciput Should I send all the activity_type with activity_type's colour from the back-end?